salt/modules/rpm.py

```diff
@@ -434,12 +433,13 @@ def info(*packages):
                                                           "Version: %{VERSION}\n"
                                                           "Vendor: %{VENDOR}\n"
                                                           "Release: %{RELEASE}\n"
-                                                          "Build Date: %{BUILDTIME:date}\n"
-                                                          "Install Date: %|INSTALLTIME?{%{INSTALLTIME:date}}:{(not installed)}|\n"
+                                                          "Build Date: %{BUILDTIME}\n"
+                                                          "Install Date: %|INSTALLTIME?{%{INSTALLTIME}}:{(not installed)}|\n"
```

Comment:
This changes semantics. Please add `build_date_time_t` and `install_date_time_t` to report time as unix secs-since-epoch.